godtools/push/action.yaml

```diff
@@ -41,12 +41,18 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Push tmp docker image
-      if: ${{ inputs.push-tmp-image == 'true' && !startsWith(github.ref, 'refs/tags') }}
+    - name: Push tmp docker images
```

Comment:
tmp images are deleted after 30 days, right?
are rest of images kept forever? (like `my-image:develop` or `my-image:kopytko`)? or only tags + `develop`?

Reply:
afaik yes, only tmp images have a retention policy